runtime/include/chpl-comm-compiler-macros.h

```diff
@@ -158,7 +158,7 @@ chpl_bool chpl_is_node_local(c_nodeid_t node)
 // and then halt the current task.  (The exact behavior is dictated by 
 // chpl_error()).
 static inline
-void chpl_check_local(c_nodeid_t node, int32_t ln, const char* file, const char* error)
+void chpl_check_local(c_nodeid_t node, int32_t ln, const int32_t file, const char* error)
```

Comment:
My thought was that it didn't really matter, the `const` is still valid, we are not going to be changing that variable. Plus, it was easier for me to just change the type when doing find & replaces.